web/themes/custom/recipe_theme/src/js/recipe_theme.script.js

```diff
@@ -0,0 +1,14 @@
+import 'popper.js';
```

Comment:
This file is just an example for using Javascript with a Drupal theme.

For the time being please remove the contents and just keep the file so ESLint won't have problems.

Reply:
Done